packages/dev/intl-types-extension/index.d.ts

```diff
@@ -1,36 +1,93 @@
-/*
- * Copyright 2020 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under
- * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- * OF ANY KIND, either express or implied. See the License for the specific language
- * governing permissions and limitations under the License.
- */
+declare namespace Intl {
 
+  type UnicodeBCP47LocaleIdentifier = string;
+
+  type RelativeTimeFormatUnit =
+    | 'year' | 'years'
+    | 'quarter' | 'quarters'
+    | 'month' | 'months'
+    | 'week' | 'weeks'
+    | 'day' | 'days'
+    | 'hour' | 'hours'
+    | 'minute' | 'minutes'
+    | 'second' | 'seconds'
+    ;
+
+  type RelativeTimeFormatLocaleMatcher = 'lookup' | 'best fit';
+
+  type RelativeTimeFormatNumeric = 'always' | 'auto';
+
+  type RelativeTimeFormatStyle = 'long' | 'short' | 'narrow';
+
+  interface RelativeTimeFormatOptions {
+    localeMatcher?: RelativeTimeFormatLocaleMatcher,
+    numeric?: RelativeTimeFormatNumeric,
+    style?: RelativeTimeFormatStyle
+  }
+
+  interface ResolvedRelativeTimeFormatOptions {
+    locale: UnicodeBCP47LocaleIdentifier,
+    style: RelativeTimeFormatStyle,
+    numeric: RelativeTimeFormatNumeric,
+    numberingSystem: string
+  }
+
+  interface RelativeTimeFormatPart {
+    type: string,
+    value: string,
+    unit?: RelativeTimeFormatUnit
+  }
+
+  interface RelativeTimeFormat {
+    format(
+      value: number,
+      unit: RelativeTimeFormatUnit
+    ): string,
+
+    formatToParts(
+      value: number,
+      unit: RelativeTimeFormatUnit
+    ): RelativeTimeFormatPart[],
+
+    resolvedOptions(): ResolvedRelativeTimeFormatOptions
+  }
+
+  const RelativeTimeFormat: {
```

Comment:
Who is consuming this variable?

Reply:
I just copied in all of Intl https://github.com/microsoft/TypeScript/blob/master/src/lib/es2020.intl.d.ts so that we'll match when we upgrade typescript